boards/arduino/portenta_h7/Kconfig.defconfig

```diff
@@ -3,6 +3,9 @@
 
 if BOARD_ARDUINO_PORTENTA_H7
 
+config REGULATOR
+	default y if NETWORKING
```

Comment:
this looks wrong: device driver relying on regulator to be used should select this option

Reply:
Networking does not directly depend on regulators, but some device driver. This driver should be the one selecting regulator enablement. The reality though is that regulator usage is nowadays not great, we have no infrastructure for automatically turning them on/off on a device level. So maybe the defconfig should be linked to the device itself, at least future refactors will be easier to spot what's going on.